deploy/README.md

```diff
@@ -129,6 +101,11 @@ Then simply run the `deploy.sh` script. You will need to follow the
 instructions during the script as it will pause waiting for the successful
 completion at a couple of steps.
 
+## Multi-cloud support
+
+It is possible to deploy one intance of SAF and attach multiple openstack clouds
+to it. See the [Multicloud README](README-multicloud.md) for details.
```

Comment:
I guess pick _Multi-cloud_ or _Multicloud_

Reply:
And I agree that I'm being far too picky for what is likely temporary documentation :)